app/serializers/school_serializer.rb

```diff
@@ -17,7 +17,12 @@ class AttributesSerializer < Blueprinter::Base
       expressions_of_interest?(school)
     end
     field :created_at
-    field :updated_at
+    field(:updated_at) do |school|
+      [
+        school.api_updated_at,
```

Comment:
can we only use `api_updated_at` please? as anything required to be updated is pushed there regardless?